src/Moneybox.App/Features/MoveMoneyHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Moneybox.App.Domain.Services;
+using static Moneybox.App.Account;
+
+namespace Moneybox.App.Features
+{
+    public class MoveMoneyHelper
```

Comment:
This class is the main reason we felt that the the test objectives weren't met. It's great that it's re-usable and can be called from both features, however this class is another abstraction which is unecessary if the logic truly lies in the domain.  Unfortunately this isn't part of the domain - it's in the features folder, it's almost like a service but ultimately, the Account domain object doesn't contain all the logic it should

Reply:
I do understand where you are coming from with this. And take your point that MoveMoneyHelper could live beside the domain objects

I felt the abstraction was necessary (in both attempts) as this is an implementation that transcends a single domain object. Goes back to my previous comment of making sure one can actually credit *and* debit the payer and payee accounts, before calling credit or debit on one of them, only to have to go back and undo that transaction somehow if one half throws.